MapboxVisionAR/src/main/java/com/mapbox/vision/ar/view/gl/VisionArView.kt

```diff
@@ -55,11 +90,32 @@ class VisionArView : GLSurfaceView, VideoSourceListener, VisionArEventsListener
         render.onNewBackground(rgbaBytes)
     }
 
+    @Deprecated("Will be removed in 0.9.0")
     override fun onNewCameraParameters(cameraParameters: CameraParameters) {
         // TODO change render
     }
 
     fun setLaneVisualParams(laneVisualParams: LaneVisualParams) {
         render.onNewLaneVisualParams(laneVisualParams)
     }
+
+    private inner class ArViewSupporter : VideoSourceListener, VisionArEventsListener {
```

Comment:
Suggest name like `Listener`, it's private here, no need to add ArView internally